datafusion/common/Cargo.toml

```diff
@@ -40,13 +40,13 @@ pyarrow = ["pyo3", "arrow/pyarrow"]
 
 [dependencies]
 apache-avro = { version = "0.14", default-features = false, features = ["snappy"], optional = true }
-arrow = { workspace = true, default-features = false }
-arrow-array = { version = "35.0.0", default-features = false, features = ["chrono-tz"] }
+arrow = { workspace = true }
+arrow-array = { workspace = true }
 chrono = { version = "0.4", default-features = false }
 cranelift-module = { version = "0.92.0", optional = true }
 num_cpus = "1.13.0"
 object_store = { version = "0.5.4", default-features = false, optional = true }
-parquet = { workspace = true, default-features = false, optional = true }
+parquet = { workspace = true, optional = true }
```

Comment:
default-features doesn't do anything see https://github.com/apache/arrow-datafusion/issues/5667

Reply:
So this means we lose coverage of the non default feature path (which is why the tests need to be updated?)

Reply:
The tests need to be updated because https://github.com/apache/arrow-datafusion/pull/5603 enabled chrono-tz, but because it was linking in arrow-array 35, this didn't percolate to other crates that are still using arrow-array 34.